source/extensions/http/cache/file_system_http_cache/stats.h

```diff
@@ -27,9 +30,14 @@ namespace FileSystemHttpCache {
   GAUGE(size_limit_bytes, NeverImport)                                                             \
   GAUGE(size_limit_count, NeverImport)                                                             \
   STATNAME(cache)                                                                                  \
-  STATNAME(cache_path)
+  STATNAME(cache_path)                                                                             \
+  STATNAME(event)                                                                                  \
+  STATNAME(event_type)                                                                             \
+  STATNAME(hit)                                                                                    \
+  STATNAME(miss)
 // TODO(ravenblack): Add other stats from DESIGN.md
 
+#define BEEP boop
```

Comment:
Do you need this macro?

Reply:
Haha, oops. Missed that when cleaning up after debugging an accidental define-line-overflow. Thanks.